server/src/main/java/io/crate/analyze/TableIdentsExtractor.java

```diff
@@ -145,6 +146,11 @@ public Collection<RelationName> visitParameterSymbol(ParameterSymbol parameterSy
         public Collection<RelationName> visitSelectSymbol(SelectSymbol selectSymbol, Void context) {
             return selectSymbol.relation().accept(RELATION_TABLE_IDENT_EXTRACTOR, context);
         }
+
+        @Override
+        public Collection<RelationName> visitAlias(AliasSymbol aliasSymbol, Void context) {
+            return aliasSymbol.symbol().accept(this, context);
+        }
```

Comment:
It was just forgotten to override `aliasSymbol`. Do we need tests/changes doc entry?

Reply:
Yes, anything that gets backported and has a fixes entry should have a test case.